conans/client/tools/scm.py

```diff
@@ -120,3 +125,79 @@ def get_revision(self):
             return commit
         except Exception as e:
             raise ConanException("Unable to get git commit from %s\n%s" % (self.folder, str(e)))
+
+
+class SVN(SCMBase):
+    cmd_command = "svn"
+
+    def __init__(self, runner=None, *args, **kwargs):
+        def runner_no_strip(command):
```

Comment:
I cannot `strip` SVN output because it relies on chars at specific columns. It may be used for git too and promote this function to the common ancestor class ([here](https://github.com/conan-io/conan/pull/3192/files#diff-beff9987dc9b09cf5c8bd12c509b9e02R32)).

Reply:
Who is stripping the output? The regular ``runner``?